### DISCUSSION TITLE
Data transform -- 转换后的数据哪里看？

### DISCUSSION THREAD

**Discussion by mxye** [antvis/G2#5990] [Q&A] [answered]
之前的版本用的是antv dataset，transform以后的数据用dv.rows可以看到。这个版本的直接用chart.transform，那么转换后的数据哪里看？我经常是通过示例来理解需要的数据格式的，但是看不到转化后的数据，就不能直观理解各个Mark需要的数据格式了。比如.rect()，文档中说需要的数据是两组x和两组y，但语焉不详，到底啥格式？[x1, x2], [y1, y2] 还是[x1,x2,y1,y2]还是[x1,y1,x2,y2]，而且是左下角到右上角，还是左上角到右下角？找不到文档中的相应信息，又看不到示例中转化后的数据格式，只能靠蒙，幸运蒙对，下次换个Mark，恐怕没这个运气。

总之，最好还是能够看到示例中transform之后的数据。各位知道怎么看吗？

**Comment by hustcc** [comment, marked as answer]
有一个内置的 data transform，名字叫 log，可以在控制台打印出数据。
